src/autobind/parser.cr

```diff
@@ -390,5 +401,20 @@ module Autobind
       type = Type.to_crystal(cursor.type.canonical_type)
       "  $#{cursor.spelling} : #{type}"
     end
+
+    def validate_name(n = nil)
+      if n
+        if n =~ /\s/
+          unless n.starts_with? /[A-Z]/
```

Comment:
Hoho that's ugly man, you can check the syntax with the language internals. There is probably a better choice, but AFAIK `Crystal.format` do the job :)

Reply:
that feature doesn't appear to be documented?

Reply:
Yes, that's compiler internals. There is still an [example](https://github.com/crystal-lang/crystal/blob/master/samples/compiler/formatter_example.cr)